hack/dockerfiles/docs.Dockerfile

```diff
@@ -13,19 +13,28 @@ RUN apk add --no-cache rsync git
 WORKDIR /src
 COPY --from=docsgen /out/docsgen /usr/bin
 RUN --mount=target=/context \
-  --mount=target=.,type=tmpfs,readwrite  \
-  rsync -a /context/. . && \
-  docsgen && \
-  mkdir /out && cp -r docs/reference /out
+  --mount=target=.,type=tmpfs,readwrite <<EOT
```

Comment:
I wonder why this readwrite is here?

Reply:
I think it was added to sync `/context` with the generated docs.

Reply:
I mean what tmpfs is not readwrite already?

Reply:
Indeed, tmpfs is readwrite by default